settings.py

```diff
@@ -43,3 +48,8 @@
 MB_TO_BYTE = 1024 * 1024
 
 HYDROSHARE_VERISON = 1.19
+
+README_FILENAME = "ReadMe.md"
+README_COLUMN_MAP_PATH = './data/markdown_map.json'
+with open(README_COLUMN_MAP_PATH) as f:
+    README_COLUMN_MAP = json.load(f)
```

Comment:
so actually, can you keep the mapping json in the data folder and not have it in settings? Settings aren't for large data objects, more for settings or smaller objects.

Reply:
@cuahsimarko 
Yes, I agree. The tricky part it the get_readme() function is called deeper inside the "for statement" that loops through all row.  I just want to initialize the json object once as it is a literally static object.